core/src/storage/impls/storage_db/kvdb_sqlite.rs

```diff
@@ -470,7 +536,7 @@ where
     T: KeyValueDbTypes<ValueType = ValueType>,
 {
     fn get_impl(&self, key: &[u8]) -> Result<Option<Self::ValueType>> {
-        let (connection, _table_name, bytes_key_table_name, statements) =
+        let (connection, _table_name, _bytes_key_table_name, statements) =
```

Comment:
Ummm. Could you prepare two set of statements for KvdbSqlite? One for "table", another for "bytes_key" table. The bytes_key table only exists when the number key flag is set to true. The bytes_key_table_name always exists, likewise we need two statements..

I think you can remove all table_names from the destructure() method.